### PR TITLE
Toit.Jaguar version 1.0.5

### PR DESCRIPTION
**Pull request by LeonGungadinMogensen**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/46591)

### CODE REVIEW

File: manifests/t/Toit/Jaguar/1.0.5/Toit.Jaguar.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Toit.Jaguar
+PackageVersion: 1.0.5
+Installers:
+- Architecture: x86
```
Comment (Trenly) [inline manifests/t/Toit/Jaguar/1.0.5/Toit.Jaguar.installer.yaml:7]:
```suggestion
- Architecture: x64
```
Is this an x64 application? I see this in the logs, and it seems to install fine on an x64 sandbox:
```
Exception trying to launch executable C:\Users\Validator\AppData\Local\Programs\jaguar\jag.exe: System.AggregateException: One or more errors occurred. (The specified executable is not a valid application for this OS platform.)
```

Comment (zachcarp) [reply manifests/t/Toit/Jaguar/1.0.5/Toit.Jaguar.installer.yaml:7]: Any update on this being x64?

Comment (Trenly) [reply manifests/t/Toit/Jaguar/1.0.5/Toit.Jaguar.installer.yaml:7]: > Any update on this being x64?

Haven't heard back from PR author; I'll check out the branch and see if a separate PR will pass without issue

Comment (LeonGungadinMogensen) [reply manifests/t/Toit/Jaguar/1.0.5/Toit.Jaguar.installer.yaml:7]: Sorry for the confusion.
The x86 was inferred by wingetcreate. I have uploaded a new PR: https://github.com/microsoft/winget-pkgs/pull/46680

**Pull request closed without merging**